swagger.yaml

```diff
@@ -198,6 +198,12 @@ parameters:
     description: "Filter resource version, as returned by a previous ETag, to be validated; or '*' to skip the version check"
     in: header
     type: string
+  is-based-on:
+    name: is-based-on
+    required: false
+    description: "A population type to search on to return datasets that are associated with that population type."
+    in: query
+    type: string
```

Comment:
It might be worth adding an `example` parameter - not necessary though

Reply:
We can't do that in OpenAPI 2.0 for parameters :( I've added an e.g. to the description though.